modular_skyrat/modules/loadouts/loadout_ui/loadout_manager.dm

```diff
@@ -231,6 +232,13 @@
 
 	to_chat(owner, examine_block(composed_message))
 
+/// If only a certain species is allowed to equip this loadout item, display which
+/datum/loadout_manager/proc/display_species_restrictions(datum/loadout_item/item)
+	var/composed_message = span_boldnotice("\The [initial(item.item_path.name)] is restricted to the following species: <br>")
+	for(var/species_type in item.restricted_species)
+		composed_message += span_green("[species_type] <br>")
+
+	to_chat(owner, examine_block(composed_message))
```

Comment:
It should probably not send anything if there's nothing in `item.restricted_species`, so you could do something like this:
```suggestion
/datum/loadout_manager/proc/display_species_restrictions(datum/loadout_item/item)
	if(!length(item.restricted_species)
		return

	var/composed_message = span_boldnotice("\The [initial(item.item_path.name)] is restricted to the following species: <br>")
	for(var/species_type in item.restricted_species)
		composed_message += span_green("[species_type] <br>")

	to_chat(owner, examine_block(composed_message))
```